src/components/SizeObserver/__snapshots__/SizeObserver.test.tsx.snap

```diff
@@ -0,0 +1,19 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<SizeObserver /> renders at the default size on first render 1`] = `
+<span>
+  Other
+</span>
+`;
+
+exports[`<SizeObserver /> when its size changes renders at the corresponding size 1`] = `
+<span>
+  Other
```

Comment:
Is this the expected result?  Looking at the test, I would have expected `small`, but maybe I'm misunderstanding.